valuable/src/valuable.rs

```diff
@@ -205,6 +205,46 @@ impl Valuable for alloc::string::String {
     }
 }
 
+#[cfg(feature = "std")]
+impl Valuable for std::path::Path {
+    fn as_value(&self) -> Value<'_> {
+        struct Error;
+        static ERROR: Error = Error;
+        const MSG: &str = "path is not valid UTF-8 string";
+        impl fmt::Debug for Error {
+            fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+                fmt::Debug::fmt(&MSG, f)
+            }
+        }
+        impl fmt::Display for Error {
+            fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+                fmt::Display::fmt(&MSG, f)
+            }
+        }
+        impl std::error::Error for Error {}
+
+        match self.to_str() {
+            Some(s) => Value::String(s),
+            None => Value::Error(&ERROR),
+        }
```

Comment:
I'm not sure if this is really preferable. Ideally, we would like to call Path::display, but that would be difficult.

Reply:
We *could* consider adding `Path` as a primitive. What do you think?

Ref: #4

Reply:
might be worth having path be a primitive...i can imagine user code wanting to inspect them as paths. OTTOH, this would need  to be feature flagged for when the standard library is not in use (since tracing supports no-std + alloc)

Reply:
That seems to make sense. Added `Value::Path`.